src/common/mmo.h

```diff
@@ -993,6 +993,8 @@ enum {
 	JOB_OBORO,
 	JOB_REBELLION = 4215,
 
+	JOB_SUMMONER = 4218,
```

Comment:
This is a bit dangerous as is, it needs a matching change in `pc_db_checkid()`, so that the values between `JOB_REBELLION` and `JOB_SUMMONER` aren't valid (preferably in this commit as not to break git bisect).